### PR TITLE
SNAP-1481 : Spark UI SQL tab needs to show the SQL query

### PR DESCRIPTION
**Pull request by snappy-sachin**
## Changes proposed in this pull request

- UI SQL Tab now displays SQL Query string instead of CachedDataFrame handler in Description column.
- Queries executed from Snappy shell and spark shell will be displayed as actual query string under Description column.
- Queries executed using API will display CachedDataFrame handler description as per previous behavior.

## Patch testing

- Tested manually

## ReleaseNotes.txt changes

- SQL Tab on UI, Description column now displays actual SQL Query string executed instead of handler description text.

## Other PRs 

- None.

### CODE REVIEW

**Comment by sumwale** [comment]
@kneeraj @rishitesh I had added the string in CachedDataFrame but the problem with ParamLiteral is that it will show only the first string which was cached. Can you help @snappy-sachin to pass through the current string to CachedDataFrame.collectWithHandler? I think what needs to be done is make the queryString in CachedDataFrame a "var" and then change it with the current sqlText before returning it in SnappySession.getPlan

File: ReleaseNotes.txt [hunk 1]
```diff
@@ -7,6 +7,12 @@
 # ADD THE JIRA TICKET ID, IF APPLICABLE.                                                           #
 ####################################################################################################
 
+Release 0.9
+
+- New Features/Fixed Issues
+
+  [SNAP-1481] SQL Tab on UI, Discrption column now displays actual SQL Query string executed inseted of handler
```
Comment (sumwale) [inline ReleaseNotes.txt]: typo "inseted"

Comment (snappy-sachin) [inline ReleaseNotes.txt]: corrected.. Thanks Sumedh

**Comment by sumwale** [comment]
For now we can have manual checking will be fine. But a unit test will be good especially for the ParamLiteral case. Register a listener on SparkListenerBus in the test for SparkListenerSQLExecutionStart and verify that strings are as expected (both short and long forms). @kneeraj

File: ReleaseNotes.txt (hunk 1 above)
Comment (ashetkar) [inline ReleaseNotes.txt]: Please break the line after 100 char as has been followed.
Typo 'Discrption'

**Comment by snappy-sachin** [comment]
Run precheckin

**Comment by snappy-sachin** [comment]
Ran precheckin and no related failure. 
Merging..

**Comment by sumwale** [comment]
@kneeraj @snappy-sachin @rishitesh 

PLEASE DO NOT DO MERGES OR APPROVE WITHOUT RESPONSES TO THE COMMENTS. Usually reviewers won't have time to come back to check if no responses are provided to those, and merges done without addressing them.

There were two important points in the comments:

a) It will not work correctly for tokenized parameters. This was a MUST FIX before merge.

b) Unit test.

Now that merge is done, track above two in a separate ticket (or responses on why these points can be ignored).

**Comment by snappy-sachin** [comment]
@sumwale , 
I have created a task (_SNAP-1532_) for adding unit test but forgot to add that in this PR while merging.
I just have added new task(_SNAP-1539_) for making the queryString in CachedDataFrame a "var" and then change it with the current sqlText before returning it in SnappySession.getPlan.

**Pull request merged**